Add missing breaking change in 2023.6 release notes (#27728)
home-assistant/home-assistant.github.io: source/_posts/2023-06-07-release-20236.markdown

[@raman325]: https://github.com/raman325
[#93946]: https://github.com/home-assistant/core/pull/93946

---

> If you have zwave_js cover entities already and your device supports Window 
> Covering CC, you may see newly created cover entities. The old cover entities
> can safely be deleted but all customizations will need to be reapplied to the
> new entities.

Reviewer: ```suggestion
If you have zwave_js cover entities already and your device supports Window 
Covering CC, you may see newly created cover entities. The old cover entities
can safely be deleted, but all customizations must be reapplied to the
new entities.
```
